packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js

```diff
@@ -528,7 +549,7 @@ class ReactSixteenAdapter extends EnzymeAdapter {
       }
       if (lastComponent !== Component) {
         if (isStateful(Component)) {
-          wrappedComponent = class extends Component {}; // eslint-disable-line react/prefer-stateless-function
+          wrappedComponent = class extends Component { }; // eslint-disable-line react/prefer-stateless-function
```

Comment:
please revert unrelated formatting changes
```suggestion
          wrappedComponent = class extends Component {}; // eslint-disable-line react/prefer-stateless-function
```

Reply:
I reverted it!